cmds/tui.c

```diff
@@ -1212,8 +1231,17 @@ static void win_display_graph(struct tui_window *win, void *node)
 		}
 
 		if (width < COLS) {
-			w = COLS - width;
-			printw("%-*.*s", w, w, curr->n.name);
+			if (curr->n.loc) {
+				printw("%s", curr->n.name);
+				width += strlen(curr->n.name);
+
+				asprintf(&str_loc, " [%s:%d]", curr->n.loc->file->name, curr->n.loc->line);
+				printw("%-*.*s", COLS - width, COLS - width, str_loc);
```

Comment:
I couldn't find function to print as gray color for tui. do you have?

Reply:
You can find the following code in `utils/debug.c`
```c
#define TERM_COLOR_RESET        "\033[0m"
#define TERM_COLOR_GRAY         "\033[90m"
```
`pr_gray` is basically like.
```c
pr_out(TERM_COLOR_GRAY "Hello" TERM_COLOR_RESET);
```

Reply:
Sorry, it was wrong info. You can find the color setup at `init_colors` function that calls `init_pair`. Please see the following documents.
- https://linux.die.net/man/3/init_pair
- https://www.linuxjournal.com/content/programming-color-ncurses

Reply:
Thank you for your information!
I applied it. In case of terminal that doesn't support color change, white is used instead. 
Unfortunately, my terminal doesn't support change of color. so I couldn't see the gray color.
Could you check if it shows as gray color if your terminal support it? (of course if you have time :) )
Thank you.

Reply:
I think we need to check whether current line has focus.